content/trial_getting_started/examples/basic/aws_basics.md

```diff
@@ -100,46 +94,40 @@ The aws infrastructure blueprint is available [here](https://github.com/cloudify
 Uploading a blueprint to Cloudify can be done by direct upload or by providing the link in the code repository.
 The flow to do that is :
 
- * (1) upload the blueprint
- * (2) create a deployment from that uploaded blueprint - this generates a model in Cloudify DB
- * (3) run the install workflow for that created deployment to apply the model to the infrastructure.
+ 1. Upload the blueprint
+ 1. Create a deployment from that uploaded blueprint - this generates a model in Cloudify DB
+ 1. Run the install workflow for that created deployment to apply the model to the infrastructure.
 
 In order to perform this flow as a single unit we will use the **install command**.
 
 ```bash
-docker exec -it cfy_manager_local sh -c "cfy install https://github.com/cloudify-community/blueprint-examples/releases/download/5.0.5-9/virtual-machine.zip -n aws.yaml"
+cfy install https://github.com/cloudify-community/blueprint-examples/releases/download/5.0.5-9/virtual-machine.zip -n aws.yaml
 ```
 
-**Tip**: If Cloudify print out any error on this stage (for example, wrong credentials were provided) and deployment was created run:
+**Tip**: If the above flow returns an error on this stage (for example, wrong credentials were provided) and deployment was already created, you should remove that deployment before you run the command again. To do that run:
```

Comment:
I think we should mention , to stop the previous execution , because at some point , it might give operation retry or something in some cases, [ so it is worth mentioning ] to be able to do uninstall [ installed should be stopped first ]

Reply:
fixed